### ISSUE TITLE
AdminConfig in '/lib' is Insecure

### ISSUE THREAD

**Issue by nicholasalanbrown** [yogiben/meteor-admin#239]
Hello:

I can only get the package config to work by placing the AdminConfig object in my '/lib' code, not in server only code, but this leaves it accessible via the client (which someone embarrassingly pointed out to me during a presentation. Is there a workaround for this? I'm not the first user and need to override the admin email address.

Thanks!

**Comment by s7dhansh** [comment]
You can set:

    AdminConfig.adminEmails = ['a@b.com'];

**Comment by nicholasalanbrown** [comment]
Right, I've done that, but it needs to be in the /lib directory, which means it's accessible to the client...that seems like a security issue...

**Comment by peterwilli** [comment]
Hi @nicholasalanbrown,

What @s7dhansh means is that you can set your object as a whole (adminEmails not included) inside client/lib and then put the line `AdminConfig.adminEmails = ['a@b.com']; ` (with your email) inside server/lib. That way you keep the 'sensitive' part away from the client.

I found it a bit confusing at first too, I just discovered this by trial and error. Hope it helps!
